apps/backend/src/modules/activity-logs/activity-log.service.ts

```diff
@@ -104,6 +111,10 @@ class ActivityLogService implements Service {
 				});
 
 				createdActivityLogs.items.push(activityLog.toObject());
+
+				if (hasCommits) {
+					await this.projectService.updateLastActivityDate(projectId, date);
+				}
```

Comment:
not sure if this is a needed check here... we can't add an activity log without commits, or can we?